instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -289,29 +319,36 @@ def _wrap(cls, method_name, wrapper):
     wrapt.apply_patch(cls, method_name, wrapper)
 
 
-def _prepare(tracer, request_hook, func, handler, args, kwargs):
-    start_time = time_ns()
+def _prepare(instrumentation, request_hook, func, handler, args, kwargs):
+    instrumentation.start_time = default_timer()
```

Comment:
Do we want to be setting an attribute on the instrumentation itself and passing this around? Seems like the instrumentation has nothing to do with the actual request being made. @owais is there a way to pass around environmental information in tornado like we do in [flask](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py#L348)?

Reply:
The environmental information that uses in the flask module exists in flask.request, I don't have member like this in tornado but I write something that similar.
Instead of passing the instrumentation I passed dictionery of metrics, What do you think?